src/common/socket.rs

```diff
@@ -165,7 +165,7 @@ impl SockInner {
         }
 
         // the mio reading window is max at 64k (64 * 1024)
-        let mut buffer = [0; 65_536];
+        let mut buffer = [0; 1_0000];
```

Comment:
Why was this changed and shouldn't the comment be updated too?

Reply:
I gather it should be `0x1_0000`

Reply:
(Although I personally find `65_536` to be a lot more readable)

Reply:
@canndrew Exactly, I just forgot about the `0x` part :)

Reply:
> (Although I personally find 65_536 to be a lot more readable)

Yeah, this new clippy lint caught me by surprise -- but I do think it is more clear this way. `65_536` looks like a random number in decimal, but in hexadecimal its significance is obvious.

Reply:
I dunno, 65536 looks pretty recognizable to me. Although 64 * 1024 would be even better still. But I'll merge this if this is what clippy wants.

Reply:
We can discuss disabling the lint if you don't think it's appropriate for the crate. Personally I agree with clippy here since I don't know the powers of 2 by heart. I fixed the typo and re-pushed the code, final call is up to you.